index.js

```diff
@@ -1,17 +1,68 @@
 class SortedList {
-  constructor() {}
+  constructor() {
+    this.items = [];
+    this.length = this.items.length;
+    
+  }
 
-  add(item) {}
+  add(item) {
+   
 
-  get(pos) {}
+    this.items.push(item);
+    
+    this.items.sort(function(a, b) {
+      return a - b;
+    });
 
-  max() {}
+    this.length = this.items.length;
+     
+  }
 
-  min() {}
+  get(pos) {
+    if (pos < 0 || pos > this.length){
+      throw new Error("_OutOfBounds_");
+    }
+    return this.items[index];
 
-  sum() {}
+  }
 
-  avg() {}
+  max() {
+    if (this.length === 0){
+      throw new Error("EmptySortedList")
+    } else {
+      
+      return Math.max(...this.items)
+    }
+
+  }
+
+  min() {
+    if (this.length === 0){
+      throw new Error("EmptySortedList")
+    } else {
+      
+      return Math.min(...this.items)
+    }
+
+  }
+
+  sum() {
+    if (this.length === 0){
+      return 0
+    } else { 
+      let suma = 0;
+      this.items.forEach(function(item){suma += item;});
+      return suma
+    }
```

Comment:
Esto esta perfect!, pero si una cosa, el forEach normalmente no se usa para esto, es mas para comprobar o acceder a cierta informacion, para convertir un array en otro tipo de dato tenemos otro metodo, el reduce encajaria mas en esta accion, es decir que seria de la siguiente forma:
```suggestion
    const result = this.items.reduce((accumulator, currentValue) => accumulator + currentValue, 0);
    return result;
```
Aqui si el array estuviera vacio, al ponerle en el reduce que el valor por defecto que devuelve es un 0, por defecto si el array viene vacio te da ese valor :D